pkg/scheduler/plugins/resourcequota/resourcequota_test.go

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2021 The Volcano Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourcequota
+
+import (
+	"fmt"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/client-go/tools/record"
+	"testing"
+	scheduling "volcano.sh/apis/pkg/apis/scheduling"
+	"volcano.sh/volcano/pkg/scheduler/api"
+	"volcano.sh/volcano/pkg/scheduler/cache"
+	"volcano.sh/volcano/pkg/scheduler/conf"
+	"volcano.sh/volcano/pkg/scheduler/framework"
+)
+
+func TestResourceQuotaPlugin(t *testing.T) {
+	tests := []struct {
+		desc            string
+		expectedOutcome bool
+		job             *api.JobInfo
+	}{
+
+		{
+			desc:            "requested_resource_within_limits",
+			expectedOutcome: true,
+			job: &api.JobInfo{
+				UID:       "1",
+				Namespace: "namespace1",
+				PodGroup: &api.PodGroup{
```

Comment:
we can define a function in  `pkg/scheduler/util/test_utils.go` to generate the wanted podgroup, so that code looks more clean.